tests/nightly/core/callbacks/compress_callback/test_compress.py

```diff
@@ -1,12 +1,13 @@
 import os
 import tempfile
 
+import pytest
 import pytorch_lightning as pl
 from pytorch_lightning.callbacks.early_stopping import EarlyStopping
 
 from anomalib.config import get_configurable_parameters
 from anomalib.core.callbacks.compress import CompressModelCallback
-from tests.core.callbacks.compress_callback.dummy_lightning_model import (
+from tests.nightly.core.callbacks.compress_callback.dummy_lightning_model import (
```

Comment:
just to be inline with the recent refactor, can we move callbacks stuff to `tests.nightly.utils`?

Reply:
I think we should be refactoring all the tests to follow the directory structure of the new refactor design. I've created an issue for this https://github.com/openvinotoolkit/anomalib/issues/95